t/22-dashboard.t

```diff
@@ -194,8 +210,8 @@ for my $url (@urls) {
 # parent group overview
 $t->get_ok('/parent_group_overview/' . $test_parent->id)->status_is(200);
 check_test_parent('expanded');
-$t->element_exists('#test_result_overview_link');
-my $tests_overview = $t->tx->res->dom->find("#test_result_overview_link")->first;
+$t->element_exists('#test_result_overview_link_1', 'exists the link to tests/overview');
```

Comment:
The same counts as for the previous occurrence. (Here it should be "link to tests/overview exists" but like I said in the other comment, you can remove these checks completely.)